db/migrate/20220513184957_add_user_to_posts.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserToPosts < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :posts, :user, null: false, foreign_key: true
```

Comment:
- [ ] According to the ERD diagram, the `post` table should have `AuthorId` instead of `user_id`. Kindly add `AuthorId` and refer it to the user table like this.
- `t.references :author, foreign_key: { to_table: 'users' }    # this will create author_id and a corresponding index`

Reply:
Thank you very much @Deepakdanger. Could you clarify this for me : Should I have to add `t.references :author, foreign_key: { to_table: 'users' }` to the file `add_user_to_posts` or I should redo everything in order to have the specified output. Thank you in advance for your response.